app/assets/javascripts/visualizations/highvis/map.coffee

```diff
@@ -184,7 +210,7 @@ $ ->
               label += "<tr><td>#{field.fieldName}</td>"
               label += "<td><strong>#{dat}</strong></td></tr>"
 
-            label += "</table></div>"
+            label += '</table></div>'
```

Comment:
Actually ``"`` is correct here because it's html.

HOWEVER - it's wrong to have html in js. But the vis 2.0 specs aren't written yet so maybe we'll let this slide.